config/routes.rb

```diff
@@ -2,5 +2,6 @@
   devise_for :users
   root to: 'pages#home'
 
+  get "family_members/:id", to: "family_members#show", as: :restaurant
```

Comment:
You wrote as: :restaurant... maybe you ment as: : family_member ?

Reply:
ok, fixed that